IDR/production/3.parse_counts_data.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
```

Comment:
Consider providing a module docstring at the top of this file to describe and remind of the purpose. This is a common practice with Python files. For ex:
```suggestion
"""
This module provides capabilities x, y, z...
"""
import pandas as pd
```